app/docs/src/reference/actions.md

```diff
@@ -109,6 +110,67 @@ Refresh actions and qualifications are the only functions that can run directly
 on HEAD without going through a change set, since they don't modify resources -
 they only read current state or validate configuration.
 
+## Action Arguments
+
+Action functions take an `Input` argument. It has a `properties` field which
+contains an object that has:
+
+- The `si` properties
+
+  These are the core properties set as meta-data for the function. Name, color,
```

Comment:
I would say these are SI-specific. It's really domain that is "core". This is just all the sugar for what you've modeled or discovered or imported in SI. Arguably color is dead too outside of schemas. It's really only the name these days from what I recall.